js/components/Team/TeamTitle.js

```diff
@@ -0,0 +1,65 @@
+import Component from "../../core/component.js";
+import store from '../../store/index.js';
+import {addTeam, deleteTeam} from "../../service/TeamApi.js";
+
+export default class TeamTitle extends Component {
+    constructor(element) {
+        super({
+            store,
+            element
+        });
+    }
+
+    teamTemplate = (teamList) => {
+        const template = teamList.map((team) => {
+            return `<div class="team-card-container">
+                <a href="/kanban.html?name=${team._id}"  class="card">
+                    <div class="card-title">
+                        ${team.name}
+                    </div>
+                </a>            
+              <button class="destroy" data-id="${team._id}">X</button>
+            </div>`
+        }).join('')
+        return template;
+
+    }
```

Comment:
```suggestion
    teamTemplate = (teamList) => (
        teamList.map((team) => (
           `<div class="team-card-container">
                <a href="/kanban.html?name=${team._id}"  class="card">
                    <div class="card-title">
                        ${team.name}
                    </div>
                </a>            
              <button class="destroy" data-id="${team._id}">X</button>
            </div>`
        ).join('')
    )
```
중괄호 대신 소괄호를 써주면 리턴을 생략할 수도 있습니다!